routes/api.php

```diff
@@ -21,13 +21,15 @@
 Route::post('verify-email/resend', 'Auth\VerificationController@resend')->name('verification.resend');
 Route::post('logout', 'Auth\AuthController@logout')->name('logout')->middleware(['auth:api', 'verified']);
 
-// Metrics
-Route::get('activity/{activity}/logview', 'Api\V1\MetricsController@logview'); // Comes from consumers (see tsugi plugin). Will not be logged in
-
 Route::group(['prefix' => 'v1', 'namespace' => 'Api\V1'], function () {
     Route::get('projects/{project}/load-shared', 'ProjectController@loadShared');
     Route::get('playlists/{playlist}/load-shared', 'PlaylistController@loadShared');
 
+    // Metrics
+    Route::get('activities/{activity}/logview', 'MetricsController@activity_logview');
```

Comment:
see if "log-view" makes more sense / follows PayPal guidelines.
Maybe, discuss with Yuri, just in case.